multinet/api/tests/factories.py

```diff
@@ -22,6 +22,22 @@ class Meta:
     name = factory.fuzzy.FuzzyText()
 
 
+class PrivateWorkspaceFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Workspace
+
+    name = factory.fuzzy.FuzzyText()
+    public = False
```

Comment:
Related to my above comment, I don't think this factory is necessary, since `public` defaults to `False` (making this identical to `WorkspaceFactory`), and `PublicWorkspaceFactory` covers the other case.

Reply:
Oh yeah I can see how this was confusing. The idea was to eventually get rid of `WorkspaceFactory` and replace it with a private and a public version to be painfully clear what each one was doing. If it makes more sense to NOT have an explicity `PrivateWorkspaceFactory` I'm fine with that.

Reply:
I see. In that case, I'd say it's probably better to have `PrivateWorkspaceFactory` be the one that survives, since it's more clear that the `User` fixture _doesn't_ have access to it, and needs to explicitly be given access.